src/firecfg/main.c

```diff
@@ -171,10 +171,10 @@ static void set_file(const char *name, const char *firejail_exec) {
 	free(fname);
 }
 
-// parse /usr/lib/firejail/firecfg.cfg file
+// parse /etc/firejail/firecfg.cfg file
```

Comment:
```suggestion
// parse /etc/firejail/firecfg.config file
```

The filename is currently misspelled.

Reply:
Aha, good catch, I will clean up these misspellings of `firecfg.config` and update their paths too.

Reply:
> Aha, good catch, I will clean up these misspellings of `firecfg.config` and update their paths too.

Done, I think.

Reply:
@hlein [on Nov 6](https://github.com/netblue30/firejail/pull/4669#discussion_r744079948):

> Aha, good catch, I will clean up these misspellings of `firecfg.config` and
> update their paths too.

Thanks.

From the new commit message:

> Also fixed some firecfg.cfg -> firecfg.config references.

One minor non-blocking nitpick: I'd `s/fixed/fix/` to make it more similar to
this:

* <https://chris.beams.io/posts/git-commit/>